backstop/tests/progress-indicator.html

```diff
@@ -28,9 +28,17 @@
     />
     <!-- Margin on the bottom of each test div to space them out. -->
     <style>
-      .iui-progress-indicator-radial {
+      #demo-radial .iui-progress-indicator-radial {
         margin-right: 16px;
       }
+
+      #demo-overlay-content {
+        width: 33%;
```

Comment:
Maybe apply this `width` to `<section>` instead because now it takes full width.